Rules/CommonScripts/AdminInventoryView.as

```diff
@@ -0,0 +1,79 @@
+#define CLIENT_ONLY
+
+void onRender(CRules@ this)
+{
+	CMap@ map = getMap();
+	if (map is null) return;
+
+	CPlayer@ player = getLocalPlayer();
+	if (player is null) return;
+
+	CControls@ controls = player.getControls();
+	if (controls is null) return;
+
+	Vec2f mouse_pos = controls.getMouseWorldPos();
+
+	if (!player.isMod())
+	{
+		this.RemoveScript("AdminInventoryView.as");
+		return;
+	}
+
+	if (player.getTeamNum() == this.getSpectatorTeamNum() && controls.isKeyPressed(KEY_LCONTROL))
+	{
+		CBlob@[] targets;
+		f32 radius = 32.0f;
+
+		if (map.getBlobsInRadius(mouse_pos, radius, @targets))
+		{
+			for (int i = 0; i < targets.length(); i++)
+			{
+				CBlob@ blob = targets[i];
+				if (blob.getInventory() !is null)
+				{
+						DrawInventoryOfBlob(blob);
+						break;
+				}
+			}
+		}	
+	}
+}
+
+void DrawInventoryOfBlob(CBlob@ this)
+{
+	CInventory@ inv = this.getInventory();
+	string[] drawn;
+	Vec2f tl = this.getInterpolatedScreenPos();
+	Vec2f offset (-40, -120);
+	u8 j = 0;
```

Comment:
This variable should have a better name. It's not clear what this variable is for at first glance.